xonsh/aliases.py

```diff
@@ -402,6 +401,7 @@ def which(args, stdin=None, stdout=None, stderr=None):
                     print(arg, file=stdout)
             else:
                 print("aliases['{}'] = {}".format(arg, builtins.aliases[arg]), file=stdout)
+                builtins.__xonsh_superhelp__(builtins.aliases[arg])
```

Comment:
I think you might want to check that `aliases[arg]` is callable() before running this line.  Otherwise you will get superhelp for a list of args.